src/redismodule.h

```diff
@@ -747,6 +747,14 @@ typedef enum {
     REDISMODULE_ACL_LOG_CHANNEL /* Channel authorization failure */
 } RedisModuleACLLogEntryReason;
 
+/* This is to prevent a coupling between redis internals and modules */
+#define RM_SCRIPT_FLAG_NO_WRITES        (1ULL<<0)
```

Comment:
We should use `REDISMODULE` prefix to avoid collusions.  Like `REDISMODULE_SCRIPT_FLAG_NO_WRITES` etc.